package.json

```diff
@@ -52,9 +52,9 @@
     "karma-coverage-istanbul-reporter": "~2.1.0",
     "karma-jasmine": "~2.0.1",
     "karma-jasmine-html-reporter": "1.5.4",
-    "ng-packagr": "^15.0.3",
+    "ng-packagr": "16.2.3",
     "protractor": "~7.0.0",
```

Comment:
Are we using protractor in this library? If that's not the case, I would get rid of this dependency and if later a test automation project is created, then we should add WDIO.

Reply:
No, we don't. This has been removed